src/poliastro/plotting/tisserand.py

```diff
@@ -0,0 +1,214 @@
+""" Generates Tisserand plots """
+
+import matplotlib.pyplot as plt
+import numpy as np
+from astropy import units as u
+from matplotlib.lines import Line2D
+from poliastro.plotting._base import BODY_COLORS
+from poliastro.twobody.mean_elements import get_mean_elements
+from poliastro.util import norm
+
+
+class TisserandPlotter:
+    """ Generates Tisserand figures """
+
+    def __init__(self, axes=None):
+        """ Object initializer
+
+        Parameters
+        ----------
+        axes: ~matplotlib.pyplot.axes
+            Axes for the figure
+
+        """
+
+        # Check if axis available
+        if not axes:
+            _, self.ax = plt.subplots(1, 1)
+        else:
+            self.ax = axes
+
+        # Tisserand kind collection
+        self.type_collection = ["energy", "period", "apsis"]
+
+        # Prellocate internal data lines
+        self.energy_lines = []
+        self.period_lines = []
+        self.apsis_lines = []
+
+    @property
+    def rp_vs_ra(self):
```

Comment:
I suggest removing all the logic that mutates the axes, and instead offering something like this:

```
tisserand = TisserandPlotter()
tisserand = TisserandPlotter(kind=TisserandKind.ENERGY)  # Same as above, default kind
tisserand = TisserandPlotter(kind=TisserandKind.PERIOD)
tisserand = TisserandPlotter(kind=TisserandKind.APSIS)
```

How does that sound @jorgepiloto?

Reply:
I tried this implementation and really like it! Let me push new changes before cleaning git commit history :+1: